python/taichi/lang/ops.py

```diff
@@ -419,6 +419,9 @@ def bit_shl(a, b):
 def bit_sar(a, b):
     return _binary_operation(ti_core.expr_bit_sar, ops.rshift, a, b)
 
+@binary
+def bit_shr(a, b):
+    return _binary_operation(ti_core.expr_bit_shr, ops.rshift, a, b)
```

Comment:
There's no proper way to implement logical right shift in python scope for **ALL** possible integer widths since python has unlimited integer widths, so probably we should disable this?